### DISCUSSION TITLE
Profile problem

### DISCUSSION THREAD

**Discussion by altafbagwan7865** [community/community#33561] [Other Feature Feedback, Questions, & Ideas] [answered]
![IMG_20220920_094617](https://user-images.githubusercontent.com/113826206/191183339-55beff52-d171-4399-a961-e81480ad0ec0.jpg)
When i click to my profile ,,there showing(sorry something went wrong) plzz tell me about this problem solution

**Comment by radar-blip** [comment, marked as answer]
Does the same thing happen repeatedly?

**Reply by altafbagwan7865** [reply]
Yes sir this thinking happened repeatedly plz give me solutions

**Reply by radar-blip** [reply]
Are you trying so sign into 2 different accounts at the same time?
That might cause something to go wrong.

**Reply by altafbagwan7865** [reply]
May I only check sir but there also showing this problem only only i checking sir

**Reply by radar-blip** [reply]
Okay, well it looks like you're trying to sign on here: https://github.com/altafbagwan
But you are already signed in, but on another name.

**Reply by altafbagwan7865** [reply]
Sir i am students I am github I don't know about it what can I do next step sir can you plzz tell me

**Reply by altafbagwan7865** [reply]
I am learning github sir

**Reply by radar-blip** [reply]
Im trying. 😄 
It seems you have 2 accounts. If you remember your password(s), you could try signing out, and sign back in again.

**Reply by altafbagwan7865** [reply]
Sir can I  delete  my one account permanently and other one can I singing out and sign in sir ?

**Reply by radar-blip** [reply]
Yes you can. As long as you have your password.
To delete one of your accounts, go to settings, and in the left column click on Accounts.
At the bottom you will see this:

![delete](https://user-images.githubusercontent.com/39196767/191192873-ddc741f5-ec05-4cdc-ad8d-44b567666e34.png)

Just make sure you keep the account you want to use. And delete the other.

**Reply by versona-tech** [reply]
f ** github It does not contain support only a handful of idiots

**Comment by altafbagwan7865** [comment]
yes sir this problem is repeatedly

**Comment by Sagar-Sharma-7** [comment]
Try logging in with some other browser or download the desktop app of GitHub.

**Reply by altafbagwan7865** [reply]
I done this sir but also doing like this only

**Comment by touwys** [comment]
I can concur that I have had the same issue with my profile for quite some time, repeatedly. Clicking on my profile image, flags the message: "Sorry something went wrong". I have only one account. One of the problems this profile issue appears to have brought along, is that I cannot poste images to attach to my messages, and neither can I activate message preview before posting it.
